blocks/address/theme-override.d.ts

```diff
@@ -0,0 +1,240 @@
+import { Components } from "@mui/material";
```

Comment:
There's a task to avoid duplicating this [here](https://app.asana.com/0/1203179076056209/1203983715193924/f) – there's a potential solution I didn't have the time to progress